### ISSUE TITLE
Invalid redirects when INCePTION is hosted inside a Docker container behind a reverse proxy

### ISSUE THREAD

**Issue by mkranzlein** [inception-project/inception#4071]
### Describe the bug

I have Apache installed on a server and INCEpTION running inside a Docker container on that same server. INCEpTION is running on container port 8080 which is mapped to host port 9332. I'm trying to use Apache to reverse proxy traffic such that `example.com/inception` goes to `localhost:9332` (and therefore the container with INCEpTION).

When I connect to `example.com/inception` in my browser, I immediately get redirected to `example.com/login.html` (INCEpTION's login page). This tells me that my request is being properly routed through to INCEpTION inside of the Docker container. However, because the redirect is going to `example.com/login.html` instead of `example.com/inception/login.html`, I get a 404 error.

I have tried tinkering with the `settings.properties` file as described in the [docs](https://inception-project.github.io/releases/23.7/docs/admin-guide.html#sect_reverse_proxy) and with the reverse proxy setup, but nothing I've tried so far has fixed the issue.

Currently, my `settings.properties` file looks like this:

```
# Port INCEpTION is listening on
server.port=8080

# If your reverse proxy is running on the same host as {product-name},
# you can use the next line to prevent direct access to INCEpTION from other hosts
# server.address=127.0.0.1

# In our examples, we run {product-name} at `your.public.domain.name.com/inception`
# If you want to Run {product-name} directly under the host name without an
# additional path, remove this line
server.servlet.context-path=/inception

# Tell {product-name} which URL your users will enter into their browsers to access it.
# Make sure you have an entry with and an entry without the protocol.
# If you also allow unencrypted http (not recommended) then also add a line with
# the http protocol
wicket.core.csrf.accepted-origins[0]=example.com
wicket.core.csrf.accepted-origins[1]=https://example.com
```

And my Apache `<Location>` directive looks like this:
```
<Location /inception>
Order allow,deny
Allow from all
AllowOverride All

ProxyPass http://localhost:9332/inception disablereuse=On
ProxyPassReverse http://example.com/inception
</Location>
```

In essence, I'm trying to do what's described in both of these sections of the docs at the same time:
- [Running behind a reverse proxy](https://inception-project.github.io/releases/23.7/docs/admin-guide.html#sect_reverse_proxy)
- [Running via Docker](https://inception-project.github.io/releases/23.7/docs/admin-guide.html#sect_docker)

### To Reproduce

1. Set up INCEpTION inside of Docker container according to [Running via Docker](https://inception-project.github.io/releases/23.7/docs/admin-guide.html#sect_docker)
2. Set up reverse proxy following [Running behind a reverse proxy](https://inception-project.github.io/releases/23.7/docs/admin-guide.html#sect_reverse_proxy)
3. Navigate to `example.com/inception`

### Expected behavior

Should take you to the login page for INCepTION.

### Screenshots

![image](https://github.com/inception-project/inception/assets/8162250/14f5b059-b220-41ac-a1cb-1b73cbe7e774)

**Comment by reckart** [comment]
Are you sure that INCEpTION reads your `settings.properties` file and that it takes effect?

When you start up INCEpTION, one of the first lines logged is the location of the `settings.properties` file and if that file `exists` (which usually means it is also read). 

Which INCEpTION version are you using?

You might try adding the line `server.forward-headers-strategy=NATIVE` in your `settings.properties`- although I believe normally it should not be necessary.

**Comment by mkranzlein** [comment]
I checked the logs, and it looks like `settings.properties` was the issue. Good catch!

I had some residual docker volumes that I cleaned up. Then I added a `.env` file to define `INCEPTION_HOME` and `INCEPTION_DB_HOME`. I deleted the old containers and reran my docker compose script. I can get to the login page now, but if I try to input the admin credentials (or no credentials) and click submit, I get taken to a different error page with the url `example.com/inception/login.html?-1.-loginForm`:

![image](https://github.com/inception-project/inception/assets/8162250/c683e0d3-1033-4b37-8c54-b5bc7f81c93c)
